libs/common-services/src/main/java/uk/gov/di/ipv/core/library/service/ConfigService.java

```diff
@@ -288,7 +288,7 @@ public Map<String, ContraIndicatorConfig> getContraIndicatorConfigMap() {
             }
             return configMap;
         } catch (JsonProcessingException e) {
-            LOGGER.error("Failed to parse contra-indicator config");
+            LOGGER.error("Failed to parse contra-indicator config: {}", e.getMessage());
```

Comment:
Do you think this error message will ever include details of our CIMIT configuration?

Reply:
It does, I didn't realise we're keeping them secret, but that would make sense! I'll revert this edit

Reply:
Yeah, it's one of those slightly fuzzy lines - we could definitely log it if it's useful (since access to the logs is controlled), but in general it's good to avoid logging more sensitive info than we need.